components/Common/Search/index.module.css

```diff
@@ -29,6 +29,8 @@
     absolute
     right-2
     ml-2
+    transition-opacity
+    duration-500
```

Comment:
```suggestion
    motion-safe:transition-opacity
    motion-safe:duration-200
```

It may be better to use `safe-motion` for users who are sensitive to this type of interaction. IMO 500 ms is a little bit long 😄